config-controller/config/crd/bases/config.stackrox.io_securitypolicies.yaml

```diff
@@ -0,0 +1,200 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.15.0
+  name: securitypolicies.config.stackrox.io
+spec:
+  group: config.stackrox.io
+  names:
+    kind: SecurityPolicy
+    listKind: SecurityPolicyList
+    plural: securitypolicies
+    singular: securitypolicy
+  scope: Namespaced
+  versions:
+  - name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        description: SecurityPolicy is the Schema for the policies API
+        properties:
+          apiVersion:
+            description: |-
+              APIVersion defines the versioned schema of this representation of an object.
+              Servers should convert recognized schemas to the latest internal value, and
+              may reject unrecognized values.
+              More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources
+            type: string
+          kind:
+            description: |-
+              Kind is a string value representing the REST resource this object represents.
+              Servers may infer this from the endpoint the client submits requests to.
+              Cannot be updated.
+              In CamelCase.
+              More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: SecurityPolicySpec defines the desired state of SecurityPolicy
+            properties:
+              categories:
+                items:
+                  type: string
+                type: array
+              criteriaLocked:
+                type: boolean
+              description:
+                type: string
+              disabled:
+                type: boolean
+              enforcementActions:
+                enum:
+                - UNSET_ENFORCEMENT
+                - SCALE_TO_ZERO_ENFORCEMENT
+                - UNSATISFIABLE_NODE_CONSTRAINT_ENFORCEMENT
+                - KILL_POD_ENFORCEMENT
+                - FAIL_BUILD_ENFORCEMENT
+                - FAIL_KUBE_REQUEST_ENFORCEMENT
+                - FAIL_DEPLOYMENT_CREATE_ENFORCEMENT
+                - FAIL_DEPLOYMENT_UPDATE_ENFORCEMENT
+                items:
```

Comment:
It should be items of enum?
items:
  enum:
     -- ...

Reply:
d0002ae8c7ff0b421c09d634cf275aa5032fc346